wasm2c/README.md

```diff
@@ -127,27 +142,34 @@ The generated header file looks something like this:
 #include "wasm-rt.h"
 
 ...
-
 #ifndef WASM_RT_CORE_TYPES_DEFINED
 #define WASM_RT_CORE_TYPES_DEFINED
+
 ...
+
 #endif
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-extern void Z_fac_init(void);
-extern void Z_fac_free(void);
+typedef struct Z_fac_module_instance_t {
+  char dummy_member;
```

Comment:
This is a little sad that we need to do this.   I would imgine that all instances would share at least one common member but I guess not.

Reply:
Perhaps in the long term we will have a type for the module itself which each instance would have a pointer too?

Reply:
Yeah... GCC and clang are happy to allow an empty struct (except with `-pedantic` which we're not currently using), but MSVC rejects it as invalid C.